### PR TITLE
adding my profile to the public/data

### PR DESCRIPTION
**Pull request by philling1**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2003"></a>

### CODE REVIEW

File: public/data/philling1.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "",
+      "date": "",
+      "icon": "",
+      "color": "",
+      "description": "",
+      "url": ""
```
Comment (Panquesito7) [inline public/data/philling1.json:25]: Please remove these empty unused fields.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by imran1509 — COMMENTED** [review]
Please use the blank fields for your other links or else delete the unused fields

**Review by accodes21 — COMMENTED** [review]
Remove Unused Links rest all is good.

**Review by philling1 — COMMENTED** [review]
done thanks

File: public/data/philling1.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "phil",
+  "type": "Personal",
```
Comment (Pradumnasaraf) [inline public/data/philling1.json:3]:
```suggestion
  "name": "Phil",
  "type": "personal",
```

**Review by Pradumnasaraf — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**